volumes/vds/sql/insert/backfill_v15_from_cnt_det.sql

```diff
@@ -0,0 +1,320 @@
+            OR v.end_timestamp IS NULL --no end date
+        )
+    JOIN vds.entity_locations AS e
+        ON e.entity_id = v.vds_id
+        AND dd.datetime_15min_rounded >= e.start_timestamp
+        AND (
+            dd.datetime_15min_rounded < e.end_timestamp
+            OR e.end_timestamp IS NULL --no end date
+        )
+    LEFT JOIN vds.detector_inventory AS di ON di.uid = v.uid
+    ORDER BY v.vds_id, dd.datetime_15min_rounded
+);
+
+--add constraints to make sure data will insert smoothly
+ALTER TABLE gwolofs.old_rescu_staging
+ADD CONSTRAINT counts_15min_partitioned_pkey PRIMARY KEY (
+    division_id, vdsconfig_uid, datetime_15min_rounded
+);
+
+CREATE INDEX IF NOT EXISTS counts_15min_div2_datetime_15min_idx
+ON gwolofs.old_rescu_staging USING brin (datetime_15min_rounded);
+
+CREATE INDEX IF NOT EXISTS counts_15min_div2_vdsconfig_uid_datetime_15min_idx
+ON gwolofs.old_rescu_staging USING btree (
+    vdsconfig_uid ASC NULLS LAST, datetime_15min ASC NULLS LAST
+);
+
+
+--compare the new and old data during overlapping period! (count_15min vs vds_counts_15min_count)
+DROP TABLE IF EXISTS gwolofs.old_rescu_compare;
+CREATE TABLE gwolofs.old_rescu_compare AS (
+    WITH ors AS (
+        SELECT
+            vdsconfig_uid,
+            datetime_15min_rounded::date AS dt,
+            SUM(count_15min) AS flow_sum
+        FROM gwolofs.old_rescu_staging
+        WHERE
+            datetime_15min_rounded >= '2017-01-01'
+            AND datetime_15min_rounded < '2021-11-01'
+        GROUP BY
+            vdsconfig_uid,
+            datetime_15min_rounded::date
+    ),
+
+    c15 AS (
+        SELECT
+            vdsconfig_uid,
+            datetime_15min::date AS dt,
+            SUM(count_15min) AS vds_sum
+        FROM vds.counts_15min_div2
+        WHERE
+            datetime_15min >= '2017-01-01'
+            AND datetime_15min < '2021-11-01'
+            AND vdsconfig_uid IN (
+                SELECT DISTINCT vdsconfig_uid FROM ors
+            )
+        GROUP BY
+            vdsconfig_uid,
+            dt
+    )
+
+    SELECT
+        COALESCE(ors.vdsconfig_uid, c15.vdsconfig_uid) AS vdsconfig_uid,
+        COALESCE(ors.dt, c15.dt) AS dt,
+        ors.flow_sum,
+        c15.vds_sum
+    FROM ors
+    FULL JOIN c15 USING (vdsconfig_uid, dt)
+);
+
+--where both have volumes, >99% are within 1%. 
+SELECT
+    --vdsconfig_uid,
+    COUNT(*) FILTER (WHERE flow_sum / vds_sum >= '0.99' AND flow_sum / vds_sum < '1.01')
+    AS within_1percent, --77,949
+    COUNT(*) FILTER (WHERE NOT (flow_sum / vds_sum >= '0.99' AND flow_sum / vds_sum < '1.01'))
+    AS outside_1percent, --373
+    COUNT(*) FILTER (WHERE flow_sum IS NULL AND vds_sum IS NOT NULL) AS flow_missing, --91,157
+    COUNT(*) FILTER (WHERE vds_sum IS NULL AND flow_sum IS NOT NULL) AS vds_missing, --5,867
+    COUNT(*) AS total_count --175,336
+FROM gwolofs.old_rescu_compare
+--GROUP BY vdsconfig_uid
+
+--create new partitions:
+SELECT vds.partition_vds_yyyy(
```

Comment:
looking at the indicies for raw_vdsdata_div_2, would we query for volume_uid?  As well as one for dt, the timestamp seems specific, is it being used in aggregation instead of datetime_15min?